common/src/main/java/com/example/android/uamp/media/MusicService.kt

```diff
@@ -206,6 +207,15 @@ open class MusicService : MediaLibraryService() {
             }
             build()
         }
+        val customActionButton =
+            CommandButton.Builder()
+                .setEnabled(true)
+                .setSessionCommand(customSessionCommand)
+                .setDisplayName("Custom command")
+                .setIconResId(android.R.drawable.ic_dialog_info)
+                .build()
+
+        mediaSession.setCustomLayout(listOf(customActionButton)) // adds custom actions in media session
```

Comment:
Instead of checking for Wear OS specifically, let's actually configure the custom action to be broadcast to all client apps. For that, I believe you should just need to call [this](https://developer.android.com/reference/androidx/media3/session/MediaSession#setCustomLayout(java.util.List%3Candroidx.media3.session.CommandButton%3E)) version of `setCustomLayout()` (i.e. with just the `CommandButton` list and no `ControllerInfo`) in `onPostConnect()`. This section of the docs might be helpful https://developer.android.com/media/implement/surfaces/mobile#config-action-buttons since adding commands for the system media controls on mobile devices is similar.